index.html

```diff
@@ -3383,24 +3386,29 @@ <h3>Verify Proof Sets and Chains</h3>
               </li>
               <li>
 Let |unsecuredDocument| be a copy of |securedDocument| with the proof value
-removed and then set |unsecuredDocument.proof| to |matchingProofs|.
-              </li>
-              <li>
-Run steps 5 through 8 to validate the contents of |proof|.
+removed and then set |unsecuredDocument|.|proof| to |matchingProofs|.
               </li>
               <li>
-Run steps 10 through 15 of the algorithm in section [[[#verify-proof]]]; if no
-exceptions are raised, associate the |isProofVerified| value with this |proof|.
-                <p class="note">
-We specifically use the value of |unsecuredDocument| as set above, rather than
-using the value from step 5 of the algorithm in section [[[#verify-proof]]].
-                </p>
+Run steps 4 through 8 of the algorithm in section [[[#verify-proof]]] on
+|unsecuredDocument|; if no exceptions are raised, add |cryptosuiteVerificationResult|
+to |verificationResults|.
                 </li>
             </ol>
           </li>
           <li>
-Return the |allProofs| along with each proof's associated
-|isProofVerified| information.
+Let |combinedVerificationResult| be `true`.
+          </li>
+          <li>
+For each |cryptosuiteVerificationResult| in |verificationResults|:
+            <ol class="algorithm">
+              <li>
+If |cryptosuiteVerificationResult|.|verified| is `false`, set |combinedVerificationResult|
+to `false`.
+              </li>
+            </ol>
+          </li>
+          <li>
+Return |combinedVerificationResult|.
```

Comment:
So I think this needs to be a [verification result](https://w3c.github.io/vc-data-model/#securing-mechanism-specifications) to comply with the general interface. I would expect this to have the overall verified status (`true` if all proofs were verified, `false` if not), and the document and media type from the last of those verifications if they all verified (otherwise `null` for each of these). The text should also say that implementations MAY return the other results and any other metadata.

Reply:
The definition of verification result you linked seems to differ from the one in this spec. Should I merge with what is in this PR (which is consistent with the rest of the document) and raise an issue, or rework the defn of verification result elsewhere in this spec to be consistent with the data model?

Reply:
We should reconcile that in another PR.

Reply:
We might as well fix the return type in this PR.

This algorithm wasn't really designed to align w/ the return type linked to, but I don't think there is any harm in doing that. We'll just want to make sure the result also includes an array of every proof that was verified so that calling software can provide an indication of which proofs succeeded and which ones did not.

Reply:
Re-ping @wes-smith -- we can't merge this PR until the above is done.